src/ui/server/helpers/string/index.test.ts

```diff
@@ -100,15 +100,15 @@ describe('server/helpers/string', () => {
 
     describe('one string is defined', () => {
       it('should return false', () => {
-        const result = stringsAreDefined('test', undefined);
+        const result = stringsAreDefined('test');
 
         expect(result).toEqual(false);
       });
     });
 
     describe('both strings are undefined', () => {
       it('should return false', () => {
-        const result = stringsAreDefined(undefined, undefined);
+        const result = stringsAreDefined();
```

Comment:
Seems that we can remove the test on line 49 with this change

Reply:
that is a different function?